### PR TITLE
feat: added landing page

### PR DESCRIPTION
**Pull request by techieshubham30**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,20 @@
...
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+     <Nav/>
+    <Routes>
+      <Route path="/" element={<Home/>} ></Route>
+      <Route path="/product" element={<ProductListing/>}></Route>
```
Comment (gopalbbharadva) [inline src/App.js:14]: Suggestion : Make one route component and render it on APP comp, it looks clean to app comp when more routes added.

**Review by gopalbbharadva — APPROVED** [review]

File: public/index.html [hunk 2]
```diff
@@ -10,34 +10,17 @@
...
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <link rel="stylesheet" href="https://bud-ui-v1.netlify.app/css/budui.css">
+    <script src="https://kit.fontawesome.com/2111f0211b.js" crossorigin="anonymous"></script>
+    
+    
     <title>React App</title>
```
Comment (kshitijthatte) [inline public/index.html:19]: You should update the title of your app

File: src/components/CategoryCard.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { Link } from "react-router-dom";
+const CategoryCard = (props) => {
+  return (
```
Comment (kshitijthatte) [inline src/components/CategoryCard.js:3]: You can destructure your props, for cleaner code

**Review by kshitijthatte — APPROVED** [review]
Good work.

File: src/components/CategoryCard.js (hunk 3 above)
Comment (techieshubham30) [reply src/components/CategoryCard.js:3]: Yes, will do that. thanks

File: src/App.js (hunk 1 above)
Comment (techieshubham30) [reply src/App.js:14]: thanks for pointing it out, will do that.

**Pull request merged**